include/stateMachineFwk/state.h

```diff
@@ -28,19 +29,48 @@ namespace SM
 	class State
 	{
   	public:
-		State(uint16_t sID);
+	        State(uint16_t sID, uint32_t duration);
```

Comment:
Keeping default value for duration can avoid lot of changes.

Reply:
Done.